src/engine/enginemaster.cpp

```diff
@@ -387,15 +387,17 @@ void EngineMaster::process(const int iBufferSize) {
 
     // Mix all the talkover enabled channels together.
     if (m_bRampingGain) {
-        ChannelMixer::mixChannelsRamping(
+        ChannelMixer::applyEffectsAndMixChannelsRamping(
```

Comment:
@daschuer I am confused by [this line](https://github.com/mixxxdj/mixxx/pull/1254/files#diff-6a1e31696d30e425b38b3ada10cf33bcR267). Why are talkover channels being added to the crossfader mix? That does not seem to fit with [your signal flow diagram](https://github.com/mixxxdj/mixxx/pull/474#issuecomment-72337890).